apps/src/applab/consoleApi.js

```diff
@@ -8,8 +8,13 @@ var consoleApi = module.exports;
 consoleApi.log = function() {
   var nativeArgs = [];
   for (var i = 0; i < arguments.length; i++) {
-    nativeArgs[i] = codegen.marshalInterpreterToNative(Applab.JSInterpreter.interpreter,
-                                                       arguments[i]);
+    if (arguments[i] === null || arguments[i] === undefined) {
+      nativeArgs[i] = arguments[i];
+    } else {
+      nativeArgs[i] =
+          codegen.marshalInterpreterToNative(Applab.JSInterpreter.interpreter,
+              arguments[i]);
+    }
```

Comment:
Another way we could write this that might be better:
```
var nativeArgs = Array.prototype.slice.apply(arguments);
for (var i = 0; i < arguments.length; i++) {
  if (arguments[i] !== null && arguments[i] !== undefined) {
    nativeArgs[i] = codegen.marshalInterpreterToNative(...);
  }
}
```
One downside is that the usage of Array.prototype.slice might be semi confusing.

Could also do the following, which might be even cleaner:
```
var nativeArgs = Array.prototype.map.apply(arguments, function (item) {
  if (item === null || item == undefined) {
    return item;
  }
  return codegen.marshalInterpreterToNative(interpreter, item);
}